include/hebi/HebiArm.m

```diff
@@ -231,6 +235,22 @@
             % Send state to module
             this.group.send(this.cmd, varargin{:});
         end
+
+        function plugin = getPluginByType(this, type)
```

Comment:
There are defined `plugin` types on [robot-config docs](https://github.com/HebiRobotics/robot-config?tab=readme-ov-file#plugins-optional) that match the 

e.g.
* GravityCompensationEffort
* DynamicsCompensationEffort
* etc.

It should compare against those and not against the arbitrary MATLAB type